app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -1062,3 +1063,11 @@ function requestNodeForceRemoval(minionId) {
     .fail(enableOrchTriggers)
     .fail(notifyRemovalError);
 }
+
+function onlineHtml(minion) {
+  if(minion.online) {
+    return '<i class="fa fa-circle text-success fa-2x" aria-hidden="true"></i>';
+  } else {
+    return '<i class="fa fa-circle text-danger fa-2x" aria-hidden="true" title="last updated at: ' + minion.updated_at + ' (admin server time)"></i>';
```

Comment:
Minor improvement adding `Offline` to the title like `Offline (last update at: ...)`.

Reply:
added